### PR TITLE
Improved UI: Edit Clinical Reminder - Admin

### LINKED ISSUE #2961: Improve gui at Patient Summary - Edit Clinical Reminder - Admin
Guessing that several small code changes can drastically improve gui at Patient Summary screen -> Edit Clinical Reminder -> Admin
(there are 2 tables on top of each other with different number of columns without any shading etc.)

image

### PR DESCRIPTION
**Pull request by thesmallstar**
Fixes #2961 

#### Changes proposed in this pull request:

- Added bootstrap button classes
- Added shading and hover to table
- Removed unwanted borders

SS:
image
dark table row is the hover effect

### CODE REVIEW

File: interface/patient_file/reminder/clinical_reminders.php [hunk 1]
```diff
@@ -76,7 +76,7 @@
 
   <div class="tab h-auto" style="width: 97%;">
     <div id='report_results'>
-      <table>
+      <table class="table table-bordered table-active table-hover">
```
Comment (tywrenn) [inline interface/patient_file/reminder/clinical_reminders.php]: Surround table with a div element that has a class of `table-responsive` ([reference](https://getbootstrap.com/docs/4.4/content/tables/#responsive-tables))

File: interface/patient_file/reminder/clinical_reminders.php [hunk 2]
```diff
@@ -132,7 +134,7 @@
       </table>
       <br />
       <br />
-      <table>
+      <table class="table table-bordered table-active table-hover">
```
Comment (tywrenn) [inline interface/patient_file/reminder/clinical_reminders.php]: Surround table with a div element that has a class of `table-responsive` ([reference](https://getbootstrap.com/docs/4.4/content/tables/#responsive-tables))

File: interface/patient_file/reminder/clinical_reminders.php (hunk 1 above)
Comment (thesmallstar) [reply interface/patient_file/reminder/clinical_reminders.php]: on it, thank you for the review :)

File: interface/patient_file/reminder/clinical_reminders.php (hunk 2 above)
Comment (thesmallstar) [reply interface/patient_file/reminder/clinical_reminders.php]: on it, thank you for the review :)

**Comment by bradymiller** [comment]
hi @thesmallstar ,This is testing well and bringing into codebase. Thanks for the code improvements! -brady

**Pull request merged**